elements/lisk-transactions/src/register_delegate.ts

```diff
@@ -26,6 +26,13 @@ export interface RegisterDelegateInputs {
 	readonly networkIdentifier: string;
 	readonly nonce: string;
 	readonly fee: string;
+	readonly senderPublicKey?: string;
+	readonly passphrases?: ReadonlyArray<string>;
+	readonly keys?: {
+		readonly mandatoryKeys: Array<Readonly<string>>;
+		readonly optionalKeys: Array<Readonly<string>>;
+		readonly numberOfSignatures: number;
```

Comment:
do we need `numberOfSignatures`?

Reply:
Since this was updated recently on your PR I need to check the impact on this one.

Reply:
Removed from sign() interface, functions and tests